src/util/XDRCereal.cpp

```diff
@@ -42,3 +45,51 @@ cereal_override(cereal::JSONOutputArchive& ar, const stellar::Asset& s,
 {
     xdr::archive(ar, stellar::assetToString(s), field);
 }
+
+void
+cereal_override(cereal::JSONOutputArchive& ar, const stellar::Memo& m,
```

Comment:
I don't think you need this override:
`Memo` contains `xstring` so the other change should take care of it.

Reply:
Yes, you're right. I started with this change, so I didn't notice that the `xstring` stuff would take care of this! Fixed.